### PR TITLE
Bump HPP to v0.5.1 and HPPO to v0.5.2

### PR DESCRIPTION
**Pull request by orenc1**
Signed-off-by: orenc1 <ocohen@redhat.com>

**Release note**:

```release-note
Bump HPP & HPPO to v0.5.0
```

### CODE REVIEW

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for this project! 

I am the bot who triggers "standard-CI" builds for this project.
As a security measure, I will not run automated tests on PRs that are not from white-listed contributors.

In order to allow automated tests to run, please ask one of the project maintainers to review the code and then do one of the following:

1. Type `ci test please` on this PR to trigger automated tests for it.
2. Type `ci add to whitelist` on this PR to trigger automated tests for it and also add you to the contributor white-list so that your future PRs will be tested automatically. ( keep in mind this list might be overwritten if the job XML is refreshed, for permanent whitelisting, please follow #3 option )
3. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the project organization, so you'll be able to run tests for all the organization's projects.

File: hack/build-manifests.sh [hunk 1]
```diff
@@ -229,7 +229,7 @@ EOM
 # Write HCO CRDs
 (cd ${PROJECT_ROOT}/tools/csv-merger/ && go build)
 hco_crds=${TEMPDIR}/hco.crds.yaml
-${PROJECT_ROOT}/tools/csv-merger/csv-merger  --api-sources=${PROJECT_ROOT}/pkg/apis/... --output-mode=CRDs > $hco_crds
+(cd ${PROJECT_ROOT} && ${PROJECT_ROOT}/tools/csv-merger/csv-merger  --api-sources=${PROJECT_ROOT}/pkg/apis/... --output-mode=CRDs > $hco_crds)
```
Comment (orenc1) [inline hack/build-manifests.sh:232]: Note this change.
Otherwise `csv-merger` exits with `cannot import absolute path`

**Comment by orenc1** [comment]
Hi @awels ,
The HPP CRD is not compliant with `apiextensions.k8s.io/v1`
Could you please check and fix it?
the relevant code is here:
https://github.com/kubevirt/hostpath-provisioner-operator/blob/master/tools/helper/helper.go#L131

Thanks

```
 The CustomResourceDefinition "hostpathprovisioners.hostpathprovisioner.kubevirt.io" is invalid: 
* spec.validation.openAPIV3Schema.properties[status].properties[conditions].items: Required value: must be specified
* spec.validation.openAPIV3Schema.properties[status].type: Required value: must not be empty for specified object fields 
```

**Comment by awels** [comment]
Ugh, forgot to sync the csv generator, the yaml is correct, let me sync them.

**Comment by awels** [comment]
https://github.com/kubevirt/hostpath-provisioner-operator/pull/79 fixes the CRD in helper.go

**Comment by awels** [comment]
https://github.com/kubevirt/hostpath-provisioner-operator/releases/tag/v0.5.1 should have it fixed. if you update to use operator 0.5.1

**Comment by orenc1** [comment]
> https://github.com/kubevirt/hostpath-provisioner-operator/releases/tag/v0.5.1 should have it fixed. if you update to use operator 0.5.1

done, thanks

**Comment by awels** [comment]
Could it be that the failures are because from v1alpha1 to v1beta1, I changed one of the fields from a string to a boolean?

**Comment by orenc1** [comment]
> Could it be that the failures are because from v1alpha1 to v1beta1, I changed one of the fields from a string to a boolean? I did remove the alpha api completely, since HCO wasn't managing it. Looks like something is looking for alpha api?

Yes, looks like the olm registry container is searching for `hostpathprovisioner.kubevirt.io/v1alpha1/HostPathProvisioner` in the bundle, and can't find it.

**Comment by awels** [comment]
I guess I am stuck keeping it around for a bit....

**Comment by awels** [comment]
https://github.com/kubevirt/hostpath-provisioner-operator/pull/82 once merged I will make new release v0.5.2

**Comment by awels** [comment]
I created new 0.5.2 release that should have alpha api in it again.

**Pull request merged**